ReactSkia/core_modules/RSkInputEventManager.cpp

```diff
@@ -151,5 +159,18 @@ void RSkInputEventManager::sendNotificationWithEventType(std::string eventType,
 }
 #endif //TARGET_OS_TV
 
+int RSkInputEventManager::registerAddListener(std::function<void(std::string,int,rnsKeyAction, rnsKey,bool)> fcb){
+  RNS_LOG_DEBUG("[registerAddListener] ");
+  callbackId_++;
+  eventCallbackMap_.insert({callbackId_,fcb});
+  return callbackId_;
+}
+
+void RSkInputEventManager::removeListener(int callbackId){
+  RNS_LOG_DEBUG("[removeListener]");
+  eventCallbackMap_.erase(callbackId_);
```

Comment:
should this be callbackId received as argument

Reply:
yes, else we don't know which one we need to delete

Reply:
maybe i was not clear in previous comment.
removeListener is receiving callbackId .. but in erase callbackId_  member is used. not the provided callbackId. So we will be erasing wrong entry when we have more than one clients